pages/data.py

```diff
@@ -63,6 +76,12 @@ def render_content(tab, store_uuids, store_trips, store_demographics):
         data = store_demographics["data"]
         columns = list(data[0].keys())
         has_perm = perm_utils.has_permission('data_demographics')
+    elif tab == 'tab-trajectories-datatable':
+        if store_trajectories == {}:
+            store_trajectories = update_store_trajectories()
```

Comment:
I am glad that you have this in a separate commit - it makes understanding the code easier. I think that this would also benefit from a short code comment on why it is different from the other tabs so that people have a rough sense without having to look at the blame immediately.